src/views/MidiView.vue

```diff
@@ -186,61 +186,65 @@ onUnmounted(() => {
               <label for="multichannel">Multichannel-to-equave</label>
             </div>
           </div>
-          <label>Settings for multichannel-to-equave mode</label>
-          <div class="control multichannel-input-container">
-            <div>
-              Center channel
-              <input
-                id="multichannel-center"
-                class="control"
-                type="number"
-                min="1"
-                max="16"
-                v-model="midi.multichannelCenter"
-              />
-            </div>
-            <div>
-              Total equaves
-              <input
-                id="multichannel-num-equaves"
-                class="control"
-                type="number"
-                min="1"
-                max="16"
-                v-model="midi.multichannelNumEquaves"
-              />
+          <template v-if="midi.multichannelToEquave">
+            <label>Settings for multichannel-to-equave mode</label>
+            <div class="control multichannel-input-container">
+              <div>
+                Center channel
+                <input
+                  id="multichannel-center"
+                  class="control"
+                  type="number"
+                  min="1"
+                  max="16"
+                  v-model="midi.multichannelCenter"
+                />
+              </div>
+              <div>
+                Total equaves
+                <input
+                  id="multichannel-num-equaves"
+                  class="control"
+                  type="number"
+                  min="1"
+                  max="16"
+                  v-model="midi.multichannelNumEquaves"
+                />
+              </div>
+              <div>
+                Equaves down
+                <input
+                  id="multichannel-equaves-down"
+                  class="control"
+                  type="number"
+                  min="0"
+                  max="15"
+                  v-model="midi.multichannelEquavesDown"
+                />
+              </div>
             </div>
-            <div>
-              Equaves down
-              <input
-                id="multichannel-equaves-down"
-                class="control"
-                type="number"
-                min="0"
-                max="15"
-                v-model="midi.multichannelEquavesDown"
-              />
+          </template>
+          <template v-if="!midi.multichannelToEquave">
```

Comment:
This could be a `v-else` but then the template becomes more rigid. Maybe two separate `v-if`s is better. Just wanted to point out the syntax.

Reply:
thanks, I noticed that possibility this morning while looking around other places where v-if pops up